server/emails/templates/CollectionCreatedEmail.tsx

```diff
@@ -57,6 +57,21 @@ export default class CollectionCreatedEmail extends BaseEmail<
     return `${collection.user.name} created a collection`;
   }
 
+  protected markup({ teamUrl, collection }: Props) {
```

Comment:
This has ended up pretty repetitive, it makes me want to abstract just a touch more – why not pass `potentialAction` to `EmailLayout` as a prop and have it construct the rest?

Reply:
Don't know, it sounded weird to pass the `potentialAction` as a prop, markup seemed more in-line, semantically. But can do, if you insist :smile:

Reply:
`goToAction` 👍